modules/ROOT/examples/live-demos/math/index.html

```diff
@@ -0,0 +1,63 @@
+<textarea id="math_demo">
+  <p style="text-align: center;">The following equations are represented in the HTML source code as LaTeX/MathML expressions.</p>
+
+  <div style="text-align: center;">
+	<h1>LaTex Example</h1>
+	<aside style="background-color: #f9f9f9; border-left: 5px solid #ccc; padding: 10px; margin: 10px 0; font-style: italic; color: #555;">
+	  The following example is a representation of a LaTex expression after it was inserted as a block element after selecting "Text above and below" from the text wrap toggle.
+	</aside>
+	<tiny-math-block formula="\sum_{
+		  \begin{subarray}{l}
+			 i\in\Lambda\\
+			 0<j<n
+		  \end{subarray}}"></tiny-math-block>
+	<p>&nbsp;</p>
+  </div>
+
+  <div style="text-align: center;">
+	<h1>MathML Example</h1>
+	<aside style="background-color: #f9f9f9; border-left: 5px solid #ccc; padding: 10px; margin: 10px 0; font-style: italic; color: #555;">
+	  The following example is a representation of a MathML expression after it was inserted as an inline element. 
+	  <tiny-math-inline>
+		<math>
+		  <mrow>
+			<munder>
+			  <mo movablelimits="false">lim</mo>
+			  <mrow>
+				<mi>n</mi>
+				<mo>→</mo>
+				<mi>N</mi>
+			  </mrow>
+			</munder>
+			<mrow>
+			  <mo>(</mo>
+			  <mrow>
+				<mn>1</mn>
+				<mo>+</mo>
+				<mfrac>
+				  <mn>1</mn>
+				  <mi>n</mi>
+				</mfrac>
+			  </mrow>
+			  <mo>)</mo>
+			</mrow>
+		  </mrow>
+		</math>
+	  </tiny-math-inline>
+	  This was done by selecting "Inline with Text" from the text wrap toggle.
+	  <p>&nbsp;</p>
+	</aside>
+
+	<p style="text-align: center;">This limit represents a common expression in calculus where <code>\(n\)</code> approaches a specified value <code>\(N\)</code>.</p>
+
+	<p style="text-align: center;">To understand this expression, let's consider what happens as <code>\(n\)</code> increases:</p>
+
+	<ul style="text-align: center;">
+	  <li>When <code>\(n\)</code> is large, the term <code>\(\frac{1}{n}\)</code> tends to zero, which implies that <code>\((1 + \frac{1}{n})\)</code> approaches 1.</li>
+	  <li>If <code>\(N\)</code> represents infinity, the limit is also 1, because the additional term <code>\(\frac{1}{n}\)</code> becomes negligible.</li>
+	  <li>If <code>\(N\)</code> is a finite value, the limit would be <code>\((1 + \frac{1}{N})\)</code>. This reflects the behavior as <code>\(n\)</code> approaches a specific finite number.</li>
+	</ul>
+
+	<p style="text-align: center;">Therefore, without further context about what <code>\(N\)</code> represents, the specific result of the limit depends on whether <code>\(N\)</code> is infinite or finite.</p>
```

Comment:
Is this section necessary? seems irrelevant to the plugin functionality.

Reply:
The content in this demo, will be replaced with an updated version as requested by product.